roadmap.md

```diff
@@ -1,33 +1,34 @@
-# Roadmap
+# Roadmap and current private preview features
 
 This feature list is a peek into our roadmap. It identifies some of the significant features we are currently working on and a rough timeframe for when you can expect to see them. It is not comprehensive but is intended to provide some visibility into key investments.
 
-These features and dates are the current plans at this time and are subject to change. The **Area** column reflects the area of the product the feature aligns with most, and **State**  indicates if the feature will be in public preview or generally available.  
+These features and dates are the current plans at this time and are subject to change. The **Area** column reflects the area of the product the feature aligns with most, and **State**  indicates if the feature will be in public preview or generally available.  Links are provided to Azure DevOps (ADO) tracking that is internal to Microsoft.
 
 ### Currently in Private Preview
-| Area    | State          | Feature                                                |
-| ------- | -------------- | ------------------------------------------------------ |
-| Calling | Private Preview | .NET client library for Calling               |
-| Calling | Private Preview | [Anonymous interoperability with Microsoft Teams](https://docs.microsoft.com/azure/communication-services/concepts/voice-video-calling/teams-interop)   |
+
+| Area    | ADO |Feature                                                |
+| ------- |  ----| ------------------------------------------------------ |
+| Calling | [2093854](https://skype.visualstudio.com/SPOOL/_workitems/edit/2093854) |.NET client library for Calling              |
```

Comment:
Is it actually safe to give customers link to ADO? will they be able to open it? Just FYI we track .Net under this one https://skype.visualstudio.com/SPOOL/_workitems/edit/2211519